.github/workflows/ci.yml

```
@@ -83,3 +116,158 @@ jobs:
+      - name: Install dependencies
+        run: pnpm install --frozen-lockfile
+      - name: Download build artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: build-artifacts
+          path: packages
+      - name: Run web tests
+        run: pnpm --filter @paper-tools/web test
+
+  web-e2e-noop:
+    name: Web e2e verification (skip: no web changes)
```

actionlint (1.7.11)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)